### PR TITLE
[Merged by Bors] - chore: remove upstreamed lemmas

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/List/Basic.lean [hunk 1]
```diff
@@ -3752,14 +3752,8 @@ end Diff
 
 /-! ### enum -/
 
-theorem length_enumFrom : ∀ (n) (l : List α), length (enumFrom n l) = length l
```
Comment (eric-wieser) [inline Mathlib/Data/List/Basic.lean:3755]: What happened with the naming convention here? Has Std taken on a different scheme to mathlib?

Comment (semorrison) [reply Mathlib/Data/List/Basic.lean:3755]: The "new" statement is `(enumFrom n l).length = l.length`. The naming has always been variable here.

Comment (semorrison) [reply Mathlib/Data/List/Basic.lean:3755]: There are plenty of examples of both alternatives all over Mathlib and Std; I agree we should do something about it, but not in this PR. :-)

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**